### PR TITLE
Add documentation for administration scripts

### PR DESCRIPTION
**Pull request by pallix**

### CODE REVIEW

File: docs/administration.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
+Administration
+==============
+
+Several console scripts are provided to facilitate the administration of
+Adhocracy 3.
+
+Workflow state
+--------------
```
Comment (xi) [inline docs/administration.rst]: I would prefer to extend the help that is given on `./bin/set_workflow_state -h` and keeping this section short.

**Comment by pallix** [comment]
In my opinion it would be too much documentation for the `--help` option. I rewrote the documentation in the script to make it clearer.

**Comment by joka** [comment]
nice, but we have now documentation for these scripts at two different places. This is more work and 
its quite risky that one place will have outdated information in the future.

I also suggest to move this to the docstring / help options.
In  administration.rst we should just  just references the script functions docstrings and or the script module docstring.

**Comment by pallix** [comment]
@joka can you merge?

File: src/adhocracy_core/adhocracy_core/scripts/set_workflow_state.py [hunk 2]
```diff
@@ -16,14 +16,39 @@
 
 
 def set_workflow_state():  # pragma: no cover
-    """Set a workflow state for a given resource.
+    """Set a workflow state for a given resource."""
+    epilog = """
+Below are some usages examples. We assume there is process
```
Comment (xi) [inline src/adhocracy_core/adhocracy_core/scripts/set_workflow_state.py]: +a+ process

File: src/adhocracy_core/adhocracy_core/scripts/set_workflow_state.py [hunk 3]
```diff
@@ -16,14 +16,39 @@
...
+is entered::
 
-        bin/set_workflow_state etc/development.ini <resource-path> <state>
-    """
+    ./bin/set_workflow_state etc/development.ini
+    /organisation/workshop evaluate result closed
+
+An absolute path can be given instead of a relative one with the
+`absolute` option. The following command will put the workflow in the
+'closed' phase, whatever the current state is::
```
Comment (xi) [inline src/adhocracy_core/adhocracy_core/scripts/set_workflow_state.py]: replace 'phase' by 'state'

File: src/adhocracy_core/adhocracy_core/scripts/set_workflow_state.py [hunk 4]
```diff
@@ -16,14 +16,39 @@
...
+
+    ./bin/set_workflow_state --absolute etc/development.ini /organisation/workshop announce participate evaluate result closed
+
+The current state and information about the workflow can be obtained
+with the `info` option::
+
+    ./bin/set_workflow_state --info etc/development.ini /organisation/workshop
+
+To `reset` option is used to reset the workflow before setting the state::
```
Comment (xi) [inline src/adhocracy_core/adhocracy_core/scripts/set_workflow_state.py]: replace 'To' by 'the'

**Comment by xi** [comment]
LGTM.

**Pull request merged**